src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -77,13 +77,14 @@ const SelectPanel = props => {
   return (
     <>
       <AnchoredOverlay
+        // @ts-ignore todo
```

Comment:
adding a ts-ignore temporarily until I declare types for props properly in a follow up PR :)